sbi/neural_nets/flow.py

```diff
@@ -777,10 +777,7 @@ def build_zuko_flow(
             )
 
         # Combine transforms.
-        neural_net = zuko.flows.Flow(
-            transform,
-            flow_built.base,
-        )  # pyright: ignore[reportArgumentType]
+        neural_net = zuko.flows.Flow(transform, flow_built.base)  # pyright: ignore[reportArgumentType]
```

Comment:
Please remove all the pyright `ignore[reportArgumentType]` once #1099 is merged. This pull request will fix those errors.